cpp/dolfinx/mesh/TopologyComputation.cpp

```diff
@@ -500,15 +509,17 @@ compute_from_map(const graph::AdjacencyList<std::int32_t>& c_d0_0,
 
 //-----------------------------------------------------------------------------
 std::tuple<std::shared_ptr<graph::AdjacencyList<std::int32_t>>,
-           std::shared_ptr<graph::AdjacencyList<std::int32_t>>, std::int32_t>
+           std::shared_ptr<graph::AdjacencyList<std::int32_t>>,
+           std::shared_ptr<common::IndexMap>,
+           std::map<std::int32_t, std::set<int>>>
```

Comment:
Make the `set` `std::int32_t` or `std::int64_t` so it's clear if it hold local for global indices.

Reply:
Neither. It is for process numbers - I guess this should be int32.

Reply:
`std::int32_t` would be good then.

Seeing `int` in this part of the code makes we wondering if the int type is correct.